### PR TITLE
currentDate extracted to parameters in calculate method

### PR DESCRIPTION
**Pull request by vmlysenkov**
@khodya check, please

### CODE REVIEW

File: src/test/java/DetermineAmountOfTimeBeforeOrAfterCourseCompletionTest.java [hunk 1]
```diff
@@ -37,7 +37,8 @@ public void shouldCalculateDaysAndHoursAfterCourseCompletionUsingLocalDate1() {
         int coursesDuration = 64;
 
         LocalDateTime endDate = DetermineCourseCompletion.getCourseCompletionDate(startDate, coursesDuration);
-        ArrayList<Integer> actual = AmountOfTimeBeforeOrAfterCourseCompletion.calculateAmountOfTimeBeforeOrAfterCourseCompletion(endDate, coursesDuration);
+        LocalDateTime currentDate = LocalDateTime.of(2022, Month.MARCH, 2, 15, 0);
```
Comment (khodya) [inline src/test/java/DetermineAmountOfTimeBeforeOrAfterCourseCompletionTest.java]: this date can be defined once in @Before method

**Review by khodya — CHANGES_REQUESTED** [review]
I like you style, now commit looks atomic and clean. Please fix small issues in comments

File: src/test/java/DetermineAmountOfTimeBeforeOrAfterCourseCompletionTest.java (hunk 1 above)
Comment (vmlysenkov) [reply src/test/java/DetermineAmountOfTimeBeforeOrAfterCourseCompletionTest.java]: @khodya please, check if I understood you right

**Review by khodya — APPROVED** [review]
Approved!

**Pull request merged**